incubator/istio/templates/_helpers.tpl

```diff
@@ -14,3 +14,29 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- /*
+Credit: @technosophos
+https://github.com/technosophos/common-chart/
+labels.standard prints the standard Helm labels.
+The standard labels are frequently used in metadata.
+*/ -}}
+{{- define "labels.standard" -}}
+app: {{template "fullname" .}}
```

Comment:
Let's use `{{ template "name" . }}` for the `app` label. See recent discussion on Slack about this. Note that you will have to update label selectors in services. See e. g. here: https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/templates/controller-service.yaml#L55

And please use consistent formatting everywhere, i. e. space after '{{' and space before `}}`.

Apply everywhere in this chart.